include/temoto_robot_manager/robot_manager_interface.h

```diff
@@ -62,21 +62,26 @@ class RobotManagerInterface : public temoto_core::BaseSubsystem
         nh_.serviceClient<temoto_robot_manager::RobotExecute>(robot_manager::srv_name::SERVER_EXECUTE);
     client_viz_info_ =
         nh_.serviceClient<temoto_robot_manager::RobotGetVizInfo>(robot_manager::srv_name::SERVER_GET_VIZ_INFO);
-    client_set_target_ =
-        nh_.serviceClient<temoto_robot_manager::RobotSetTarget>(robot_manager::srv_name::SERVER_SET_TARGET);
+    client_set_manipulation_target_ =
+        nh_.serviceClient<temoto_robot_manager::RobotSetTarget>(robot_manager::srv_name::SERVER_SET_MANIPULATION_TARGET);
+    client_get_manipulation_target_ =
+        nh_.serviceClient<temoto_robot_manager::RobotGetTarget>(robot_manager::srv_name::SERVER_GET_MANIPULATION_TARGET);
+    client_navigation_goal_ =
+        nh_.serviceClient<temoto_robot_manager::RobotGoal>(robot_manager::srv_name::SERVER_NAVIGATION_GOAL);
   }
 
   void loadRobot(std::string robot_name = "")
-  {
+  {    
     std::string prefix = temoto_core::common::generateLogPrefix(log_subsys_, log_class_, __func__);
     validateInterface(prefix);
-
     // Contact the "Context Manager", pass the gesture specifier and if successful, get
     // the name of the topic
     temoto_robot_manager::RobotLoad load_srvc;
-    load_srvc.request.robot_name = robot_name;
+    load_srvc.request.robot_name = robot_name;      
     try
     {
+      TEMOTO_INFO_STREAM(robot_manager::srv_name::MANAGER);
+      TEMOTO_INFO_STREAM(robot_manager::srv_name::SERVER_LOAD);
```

Comment:
if you plan to leave these info_streams in then try making them print something more readable than just manager's and server's name (currently it's unclear why are you printing them out)